### ISSUE TITLE
Remove legacy Button Block

### ISSUE THREAD

**Issue by gsambrotta** [kendraio/kendraio-app#470]
In the code there is a `button-block.component.ts` which is not used anywhere. 
It is confusing to have it there as we do have button in the UI but they are under the Action Block.

We should clean up the code and avoid further confusion.
We should remove:
- src/app/blocks/button-block

Remove it from here:
src/app/components/blocks-workflow/blocks-workflow.component.html
```
  97:    import {ButtonBlockComponent} from './blocks/button-block/button-block.component';
```

And here:
/home/anaketa/repos/Kendraio/kendraio-app/src/app/app.module.ts
```
  15: <app-button-block class="block-wrapper" *ngSwitchCase="'button'" [context]="context" [config]="block" [model]="models[i]" (output)="updateModel(i + 1, $event)"></app-button-block>
 ```

**Comment by CodeKrakken** [comment]
Agreed - if it definitely isn't used it should go. Are you sure it's obsolete?
